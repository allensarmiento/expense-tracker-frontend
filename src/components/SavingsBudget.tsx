import BannerSection from './BannerSection';
import BannerTitle from './BannerTitle';

const SavingsBudget = () => {
  return (
    <BannerSection>
      <BannerTitle>Savings  Budget</BannerTitle>
    </BannerSection>
  );
};

export default SavingsBudget;
