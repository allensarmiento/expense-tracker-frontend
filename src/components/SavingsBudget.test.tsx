import { shallow } from 'enzyme';
import SavingsBudget from './SavingsBudget';

it('renders without crashing', () => {
  expect(shallow(
    <SavingsBudget />
  )).toMatchSnapshot();
});
