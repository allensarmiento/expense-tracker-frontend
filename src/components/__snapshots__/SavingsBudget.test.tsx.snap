// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<BannerSection>
  <BannerTitle>
    Savings Budget
  </BannerTitle>
</BannerSection>
`;
